app/models/local_authority.rb

```diff
@@ -4,4 +4,8 @@ class LocalAuthority < ActiveRecord::Base
   validates :homepage_url, format: { with: URI.regexp }, allow_blank: true
   validates :tier, inclusion: { in: %w(county district unitary),
     message: "%{value} is not an allowed tier" }
+
+  def provided_services
+    Service.for_tier(self.tier)
+  end
```

Comment:
Oh, that's how the limiting to local authority works: by tiers. I was wondering how we'd do that.